tests/test_cell_content_formatter.py

```diff
@@ -20,19 +21,45 @@ def test_variable_enrichements(self):
         ).substitute_user_variables()
         assert enriched_cell == "select * from table_name v where v.id = 1"
 
-    """It should throw NonExistentVariableException in case of undefined variable"""
+    def test_hidden_variable_enrichments(self):
+        """
+        Test Jupyter cell input text enrichment with both variables defined in
+        IPython kernel and get by getpass method
```

Comment:
I think we should split it into two tests.

Reply:
I have added comments explaining what is happening in the test and why it should not be split :)